### PR TITLE
Add refs

### PR DESCRIPTION
**Pull request by TSedlar**
Added:
- MemberInfo#classInfo
- AbstractInstruction#methodInfo

### CODE REVIEW

File: src/core/jvm/MemberInfo.js [hunk 1]
```diff
@@ -6,16 +6,26 @@ import * as _ from 'lodash';
  */
 class MemberInfo {
   /**
+   * @param  {ClassInfo} classInfo - The class the given member is in
    * @param  {Number} accessFlags - access flags bit string
    * @param  {string} name - name of this entry in the ClassFile
    */
-  constructor(accessFlags, name) {
+  constructor(classInfo, accessFlags, name) {
```
Comment (kylestev) [inline src/core/jvm/MemberInfo.js]: ClassInfo extends MemberInfo. not a fan of this. should only be available in MethodInfo and FieldInfo

**Comment by TSedlar** [comment]
Ah, my bad. In other libs I've only seen field/method extend the same class, will ammend.

**Comment by kylestev** [comment]
squashed the bug fix commit i made into 44c977e

**Comment by kylestev** [comment]
@TSedlar I really like the idea of extracting the owner/name/desc to `MemberInstruction` :+1:

**Pull request merged**